### DISCUSSION TITLE
RubyAtScale/Packwerk community

### DISCUSSION THREAD

**Discussion by iMacTia** [rubyatscale/packs-rails#17] [General]
Thank you so much for setting up `rubyatscale` and sharing all these projects with the OSS community 🙏 .
It's great to see Packwerk being used more and more by different organisations and the Packwerk ecosystem growing even larger!

The more I work with Packwerk and its ecosystem, the more I wish there was a community to help out teams embarking on this adventure. With Packwerk being an unopinionated tool, it can be hard at the beginning to understand how to incorporate it into your project. A community could also help improve the tooling and the conventions over time 🙌!

Is there something like this already that I'm missing out on? If not, would you be open to setting something up?
I'm thinking about something like what @nateberkopec has done with the Rails Performance Slack community ❤️

The focus of such a community could be specifically Packwerk, or maybe more at the "Ruby at scale" level 😄

**Comment by alexevanczuk** [comment]
Hey @iMacTia absolutely! We have a `Ruby/Rails Modularity` slack server, here's the invite link: https://join.slack.com/t/rubymod/shared_invite/zt-1dgyrxji9-sihGNX43mVh5T6tw18hFaQ

Let me know if this is what you had in mind!

**Reply by tim-gq** [reply]
This slack server has been deleted now right @alexevanczuk?

**Reply by alexevanczuk** [reply]
Yes, we removed the slack server, mostly because discourse is free – we have a discourse forum here: https://rubyrailsmodularity.com

Not super active... but it's there!
